drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -88,4 +88,34 @@ config ETH_STM32_CARRIER_CHECK_RX_IDLE_TIMEOUT_MS
 	  Set the RX idle timeout period in milliseconds after which the
 	  PHY's carrier status is re-evaluated.
 
+if SOC_SERIES_STM32F7X || SOC_SERIES_STM32H7X
```

Comment:
Oh God, it needs to be by SOC uh?

Reply:
Yeah, mostly same procedure but different registers :-(